spec/exception_spec.rb

```diff
@@ -75,6 +75,9 @@
       expect(Azure::Armrest::BadGatewayException).to_not be_nil
       expect(Azure::Armrest::GatewayTimeoutException).to_not be_nil
       expect(Azure::Armrest::TooManyRequestsException).to_not be_nil
+      expect(Azure::Armrest::PayloadTooLargeException).to_not be_nil
+      expect(Azure::Armrest::ServiceUnavailableException).to_not be_nil
+      expect(Azure::Armrest::LoopDetectedException).to_not be_nil
     end
```

Comment:
A more semantically truthy (and nicer error messages) way would be `expect(const_defined?(Azure::Armrest::Exception)).to be_truthy` (note you can also do `expect(Azure::Armrest::Exception).to be_const_defined` but I dislike the grammar bastardization here intensely...)